webapp/package.json

```diff
@@ -33,6 +33,7 @@
     "@angular/platform-browser": "^16.2.3",
     "@angular/platform-browser-dynamic": "^16.2.3",
     "@angular/router": "^16.2.3",
+    "@messageformat/core": "^3.2.0",
```

Comment:
I think the rebase had an issue because the old `messageformat` is back in this package.json below.